### PR TITLE
style: fix eslint errors

### PR DESCRIPTION
**Pull request by taylrj**
Since [this change](https://github.com/twreporter/keystone/pull/192/commits/3d2733df2f4edf880de2d92709957bd3fc350155), the mixed space and tab indent issue has been resolved by eslint[1] (maybe fixed by [2]), we can finally using `eslint --fix` to address indentation issue.

This change resolves indentation issues. Also, update .eslintrc to fix eslint errors.

[1] https://github.com/twreporter/keystone/pull/181#issuecomment-877095321
[2] https://github.com/eslint/eslint/commit/cc534819c739006a9d2680635b90fae9fbdec58e

### CODE REVIEW

File: fields/types/gcsimage/GcsImageColumn.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ var GcsImageColumn = React.createClass({
     );
 
     const iptc = _.get(value, ['iptc'], {});
-    const { byline, caption, created_time, keywords } = iptc;
```
Comment (babygoat) [inline fields/types/gcsimage/GcsImageColumn.js]: I suppose we should rewrite the `iptc.keywords` below?

**Comment by babygoat** [comment]
cc. @Aylie-Chou  You could also help review here so that we can determine whether to apply certain rules for the FE development.

File: admin/client/components/ImagesDiff.js [hunk 2]
```diff
@@ -10,127 +10,127 @@ import Rcslider from 'rc-slider';
 const bgImage = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUAQMAAAC3R49OAAAABlBMVEX5+fn///8pDrwNAAAAFElEQVQI12NgsP/AQAz+f4CBGAwAJIIdTTn0+w0AAAAASUVORK5CYII=';
```
Comment (babygoat) [inline admin/client/components/ImagesDiff.js:10]: Is there any reason why we should fix the indentation in two stages?

Comment (taylrj) [reply admin/client/components/ImagesDiff.js:10]: Just to be clear, the second stage causes by updating .eslintrc with `{ parserOptions: "ecmaVersion": 9 } `.  

I could merge them into single commit then.

**Comment by taylrj** [comment]
Try to add more context here.

Since [this change](https://github.com/twreporter/keystone/pull/192/commits/3d2733df2f4edf880de2d92709957bd3fc350155), the mixed space and tab indent issue has been resolved by eslint[1], we can finally using `eslint --fix` to address indentation issue.

[1] https://github.com/twreporter/keystone/pull/181#issuecomment-877095321

**Comment by babygoat** [comment]
Do we want to have this patch in the release? Since Ivory will take pre-production tests on keystone, I would like to have this one as well if it is planned.

**Comment by babygoat** [comment]
Before it is merged, I would like to know if there is any concern from @Aylie-Chou

File: fields/types/gcsimage/GcsImageColumn.js (hunk 1 above)
Comment (taylrj) [reply fields/types/gcsimage/GcsImageColumn.js]: Updated in https://github.com/twreporter/keystone/pull/194/commits/b424d963e085e315db4a5a5b8f70dfe485984ba2.

**Comment by taylrj** [comment]
> Do we want to have this patch in the release? Since Ivory will take pre-production tests on keystone, I would like to have this one as well if it is planned.

If this changed has been approved (no further concerns), yes we could.

After @Aylie-Chou's review, I will merge the two stages indentation fixes into one.

**Comment by babygoat** [comment]
If @Aylie-Chou doesn't have time to review today, then I might suggest we can postpone the merge to the next release. How do you(@taylrj ) think? I plan to release to staging by 4 PM.

**Comment by taylrj** [comment]
Sure, we can postpone the merge until it is approved by all the reviewers.

**Comment by Aylie-Chou** [comment]
Should we follow the eslint rule or always run eslint fix before commit in the future ?

File: commitlint.config.js [hunk 3]
```diff
@@ -1 +1 @@
-module.exports = {extends: ['@commitlint/config-conventional']};
+module.exports = { extends: ['@commitlint/config-conventional'] };
```
Comment (Aylie-Chou) [inline commitlint.config.js:1]: Is this [object-curly-spacing](https://eslint.org/docs/rules/object-curly-spacing) rule ?
If yes, should it been included in our .eslintrc rules ?

Comment (Aylie-Chou) [reply commitlint.config.js:1]: I find the object-curly-space rule been configured as always in eslint-config-keyston .eslintrc file.
Thus I'll resolve this conversation directly.

**Comment by taylrj** [comment]
> Should we follow the eslint rule or always run eslint fix before commit in the future ?
> It seems not only indent problem been fixed by eslint. Some coding conventions also styled by it. (like semicolon in the end)

Previously, we use [precommmit-hook](https://github.com/nlf/precommit-hook) to run `lint`, `validate`, `test` by its [default setting](https://github.com/nlf/precommit-hook#usage).

In https://github.com/twreporter/keystone/pull/194/commits/53f3b67a5e62dabde05c6288e080daa18835a868, it updates the config to run `lint-fix` on pre-commit.

Therefore, it will always run eslint fix automatically before commit in the future. All we need to do is to make sure the git hook is properly configured to let it run. Here is a possible [troubleshooting reference](https://github.com/typicode/husky/issues/435#issuecomment-450192796) works for my case when the pre-commit hook not executing.

File: admin/client/components/ImagesDiff.js (hunk 2 above)
Comment (taylrj) [reply admin/client/components/ImagesDiff.js:10]: They are squashed into https://github.com/twreporter/keystone/pull/194/commits/4de499a23b7f3b81d8f58641f7ab3ce1b0591b10.

**Review by Aylie-Chou — APPROVED** [review]

**Comment by babygoat** [comment]
LGTM

**Pull request merged**